iOS/DuckDuckGo/AIChat/InputBox/SwitchBar/SwitchBarHandler.swift

```diff
@@ -115,7 +115,6 @@ final class SwitchBarHandler: SwitchBarHandling {
     init(voiceSearchHelper: VoiceSearchHelperProtocol, storage: KeyValueStoring) {
         self.voiceSearchHelper = voiceSearchHelper
         self.storage = storage
-        restoreToggleState()
```

Comment:
I'm just deleting the restore function call because from my understanding we're still validating if this is the approach we want.